base/bootstrap.go

```diff
@@ -32,6 +35,8 @@ type BootstrapConnection interface {
 	UpdateConfig(bucket, groupID string, updateCallback func(rawBucketConfig []byte, rawBucketConfigCas uint64) (updatedConfig []byte, err error)) (newCAS uint64, err error)
 	// Close releases any long-lived connections
 	Close()
+
+	SetConnectionStringServerless() error
```

Comment:
This seems like it could be set in NewCouchbaseCluster, and wouldn't need to be modified post-initialization.  If that's accurate, does it need to be on the BootstrapConnection interface?